posthog/utils.py

```diff
@@ -595,22 +595,16 @@ def queryset_to_named_query(qs: QuerySet, prepend: str = "") -> Tuple[str, dict]
     return new_string, named_params
 
 
-def is_clickhouse_enabled() -> bool:
-    return settings.EE_AVAILABLE and settings.PRIMARY_DB == AnalyticsDBMS.CLICKHOUSE
-
-
 def get_instance_realm() -> str:
     """
-    Returns the realm for the current instance. `cloud` or 'demo' or `hosted` or `hosted-clickhouse`.
+    Returns the realm for the current instance. `cloud` or 'demo' or `hosted-clickhouse`.
```

Comment:
```suggestion
    Returns the realm for the current instance. `cloud` or 'demo' or `hosted-clickhouse`.
    
    Historically this would also have returned `hosted` for hosted postgresql based installations
```

Note this loses important historic context for someone looking through what property values exist.